src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IForLoopStatement.vb

```diff
@@ -7609,5 +7609,355 @@ Block[B6] - Exit
 
             VerifyFlowGraphAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedFlowGraph, expectedDiagnostics)
         End Sub
+
+        <CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)>
+        <Fact()>
+        Public Sub ForToFlow_38()
+            Dim source = <![CDATA[
+Imports System
+Public Class C
+    Sub M(i As Object, init As Object, limit As Object, result As Object) 'BIND:"Sub M"
+        For i = init To limit
+            result = i
+        Next
+    End Sub
+End Class
+]]>.Value
+
+            Dim compilation = CreateCompilationWithMscorlib45(source, options:=TestOptions.ReleaseDebugDll, parseOptions:=TestOptions.RegularWithFlowAnalysisFeature)
+            compilation.MakeMemberMissing(WellKnownMember.Microsoft_VisualBasic_CompilerServices_ObjectFlowControl_ForLoopControl__ForLoopInitObj)
```

Comment:
>compilation.MakeMemberMissing(WellKnownMember.Microsoft_VisualBasic_CompilerServices_ObjectFlowControl_ForLoopControl__ForLoopInitObj) [](start = 12, length = 134)

Given that you are not referencing VB runtime, this looks unnecessary. This test also covers missing other helper.